controllers/main.go

```diff
@@ -1277,10 +1277,9 @@ func (controller *MainController) PasswordResetPost(c web.C, r *http.Request) (s
 	if !controller.IsCaptchaDone(c) {
 		session.AddFlash("You must complete the captcha.", "passwordresetError")
 		return controller.PasswordReset(c, r)
-	} else {
-		session.Values["CaptchaDone"] = false
-		c.Env["CaptchaDone"] = false
 	}
+	session.Values["CaptchaDone"] = false
+	c.Env["CaptchaDone"] = false
```

Comment:
Was this wrong? It seems unrelated to the issue.

Reply:
Not related directly but saw it while working on the task, there was a redundant else statement, anyway these two lines will run if the condition is false, so I fixed it.